server/cluster/coordinator.go

```diff
@@ -849,6 +849,11 @@ func (s *scheduleController) Stop() {
 
 func (s *scheduleController) Schedule() []*operator.Operator {
 	for i := 0; i < maxScheduleRetries; i++ {
+		select {
```

Comment:
add a comment. like: Prevent the long scheduling time from being unable to exit for a long time.